e2e/test/scenarios/question/reproductions/22230-query-builder-aggregation-filtering.cy.spec.js

```diff
@@ -0,0 +1,43 @@
+import {
+  restore,
+  popover,
+  summarize,
+  openPeopleTable,
+  visualize,
+} from "e2e/support/helpers";
+
+describe("issue 22230", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  it("should allow filtering on aggregated fields", () => {
+    openPeopleTable({ mode: "notebook" });
+
+    summarize({ mode: "notebook" });
+    popover().findByRole("option", { name: "Maximum of ..." }).click();
+    // I don't know why this can't be chained, I tried it and it didn't work.
+    popover().findByRole("option", { name: "Name" }).click();
+
+    // findByRole doesn't work here, and I don't know why.
+    cy.findByText("Pick a column to group by").click();
```

Comment:
I've tried `cy.findByRole("link", { name: "Pick a column to group by" }).click();
`, but it doesn't work 🤔 
<img width="450" alt="image" src="https://user-images.githubusercontent.com/1937582/227636739-8ee0fd77-d585-467e-9f43-e0a2761a0d36.png">